data/campaigns/Heir_To_The_Throne/scenarios/30_The_Sceptre_of_Fire.cfg

```diff
@@ -0,0 +1,1395 @@
+            message_Jeniver=_"I’ve read so much about the lost Scepter of Fire! I never though I would, well, actually help <i>find</i> it. Where do we go, Delfador!"
+            message_Seimus=_"It’s overwhelming! I never— well, Delfador, thank you for trusting me to be here with you. Where shall we start the search?"
+            message_Dosh=_"Dis cave good place for troll, but no good place for softskins. Better find your scep-tar an’ get back to sun-lit places. Which way should we start?"
+            fallback_Konrad=_"Very well — then where? How do we find it?"
+        [/companion_message]
+        [message]
+            speaker=Delfador
+            message=_"North and east. More than that, I cannot tell. Whatever beasts, monsters, and other denizens of the dark lurk ahead, I know that we shall prevail."
+        [/message]
+
+        #############################
+        # OBJECTIVES
+        #############################
+        [objectives]
+            [objective]
+                description= _ "Defeat the enemy leader"
+                condition=win
+            [/objective]
+            [objective]
+                #po: this is deliberately vague, and foreshadows Li'sar's appearance in part two of this scenario
+                description= _ "Then, fight to claim the Sceptre"
+                condition=win
+            [/objective]
+            {TURNS_RUN_OUT}
+            {OBJECTIVES_HERODEATHS}
+            [gold_carryover]
+                carryover_percentage=40
+                bonus=no
+            [/gold_carryover]
+            [note]
+                description= _ "The longer you take to defeat the enemy leader, the more time your other enemies will have to accrue gold."
+            [/note]
+        [/objectives]
+    [/event]
+
+    #############################
+    # KONRAD IS WORRIED
+    #############################
+    [event]
+        name=turn 2
+        [message]
+            speaker=Konrad {KONRAD_VARIATION concerned}
+            message=_"Delfador, you’ve told me the importance of the Sceptre of Fire. And I believe you! A powerful magical artifact, and the fabled symbol of the kings of Wesnoth. With the Sceptre in my hand even Asheviere would struggle to deny my claim of heirship."
+        [/message]
+        [message]
+            speaker=Konrad {KONRAD_VARIATION concerned}
+            message=_"Yet it pains me to know that while we delve deep under these mountains, countless people back home are suffering. Elves, men, dwarves; people we’ve missed out helping because we had to come here instead. I hope it’ll be worth it in the end."
+        [/message]
+    [/event]
+
+    #############################
+    # FORESHADOW HEIR
+    #############################
+    [event]
+        name=turn 8
+        [message]
+            speaker=Konrad
+            message=_"Delfador... other than the prophecy about the Sceptre, you haven’t been any other secrets from me “for my own good”, are you?"
```

Comment:
'been keeping any other secrets'?

Also end with 'have you', since the question starts with 'you have not'.